docs/02. getting started/09. different_url_per_locale.md

```diff
@@ -0,0 +1,27 @@
+# Different url per locale
+
+In Fork CMS 5.9.0 and up it is possible to configure different urls per locale.
+
+You can do this by replacing `site.domain` in `app/config/parameters.yml` with `site.domain.[language]`
+
+Example:
+
+    parameters:
+        database.driver:        mysql
+        …
+        site.protocol:          https
+        site.domain.en:         fork-cms.com
+        site.domain.nl:         fork-cms.be
+        site.domain.fr:         fork-cms.fr
+        site.domain.de:         fork-cms.de
+        site.default_language:  en
+        site.multilanguage:     true
+        …
+        action.rights_level:    7
+
+If there isn't any url configured for a requested language the url for the default language will be loaded.
+The domain for the current language will be available in `site.domain`    
+
+The Locale classes extending from `\Common\Locale` also have a new method `getDomain` to get the correct domain for that locale.
+
+Note: This doesn't work if the parameter `site.domain` already exists in `app/config/parameters.yml`
```

Comment:
So in essence you have to chose between

    site.domain.en:         fork-cms.com
    site.domain.nl:         fork-cms.be
    site.domain.fr:         fork-cms.fr
    site.domain.de:         fork-cms.de

Or 

    site.domain:         fork-cms.com

I would explain this in the documentation that it is either one.
When I first saw the documentation, I expected that site.domain was used as the default domain

Reply:
the default domain is based on the default language

```
site.default_language:  en
site.domain.en:         fork-cms.com
site.domain.nl:         fork-cms.be
```
if you go there in fr you'll get fork-cms.com

Reply:
How would you phrase it differently?
The docs say
```
You can do this by replacing `site.domain` in `app/config/parameters.yml` with `site.domain.[language]```
and ```If there isn't any url configured for a requested language the url for the default language will be loaded.```